src/remotepublish.cmd.js

```diff
@@ -244,7 +244,7 @@ class RemotePublishCommand extends AbstractCommand {
     }).catch((e) => {
       this.tick(10, 'failed to set up logging', true);
       this.log.error(`Remote addlogger service failed ${e}`);
-      throw new Error('Unable to set up remote logging');
+      this.log.warn('Unable to set up remote logging');
```

Comment:
I think the `log.error` above is enough. no need to log again with warn. Maybe change the log level to `warn`